includes/collection/class-actors.php

```diff
@@ -1060,4 +620,217 @@ protected static function check_legacy_key_pair( $user_id ) {
 
 		return false;
 	}
+
+	/**
+	 * Returns all Inboxes for all known remote Actors.
+	 *
+	 * @deprecated unreleased Use {@see Activitypub\Collection\Remote_Actors::get_inboxes()}
```

Comment:
```suggestion
	 * @deprecated unreleased Use {@see Remote_Actors::get_inboxes()}
```
Not sure if it's just a quirk of my IDE, but `Activitypub\Collection\Remote_Actors` without the leading backslash (`\Activitypub`) creates `Undefined namespace 'Collection'` notices. 

Since this annotation is meant for IDE traversal and `Actor` and `Remote_Actor` are in the same namespace we can just remove it from this and following annotations.